reinforcement_learning/rlclientlib/schema/OutcomeEvent.fbs

```diff
@@ -0,0 +1,12 @@
+﻿// EventHubInteraction Schema used by FlatBuffer
+
+namespace VW.Events;
+
+table OutcomeEvent {
```

Comment:
The deferred action should also be defined in the OutcomeEvent.
@ataymano do you confirm?

Reply:
Yes, it should.